### PR TITLE
Packages: Ensure that private packages do not update when publishing to npm

### PR DESCRIPTION
**Pull request by gziolo**
## Description

Changes included ensure that private packages are never updated during npm publishing. We tested [`--no-private`](https://github.com/lerna/lerna/tree/main/core/filter-options#--no-private) flag when publishing npm packages with changes for WordPress 5.9.1 release last Thursday and it worked as documented in the Lerna repository. I also included a command that always picks `patch` version updates for `npm run publish:patch` which simplifies the process.

## Testing Instructions
I executed the command `npm run publish:patch` to ensure that all non-private packages get automatic patch version updated. I bailed out early to avoid the unnecessary npm publishing.

## Screenshots 

Screenshot 2022-02-21 at 10 59 57

## Types of changes
Enhancement.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [-] My code follows the accessibility standards. 
- [-] I've tested my changes with keyboard and screen readers. 
- [-] My code has proper inline documentation. 
- [-] I've included developer documentation if appropriate. 
- [-] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). 
- [-] I've updated related schemas if appropriate.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -266,8 +266,8 @@
 		"postinstall": "patch-package && node ./patches/patch-xcode.js",
 		"prepublishOnly": "npm run clean:package-types && npm run build:packages",
 		"publish:check": "lerna updated",
-		"publish:patch": "lerna publish --dist-tag patch",
-		"publish:latest": "lerna publish",
+		"publish:patch": "lerna publish patch --no-private --dist-tag patch",
```
Comment (Mamaduka) [inline package.json:269]: Do we need to `patch` keywords for this command?

Comment (gziolo) [reply package.json:269]: `patch` just after `lerna publish` is for applying patch version update to all changed packages so you don't have to do it manually.

**Review by Mamaduka — APPROVED** [review]
Thank you, @gziolo!

File: package.json (hunk 1 above)
Comment (Mamaduka) [reply package.json:269]: Oh, that makes sense :)

**Pull request merged**